tests/client/stickers_api/test_get_sticker_packs.py

```diff
@@ -0,0 +1,187 @@
+import pytest
+import respx
+
+from botx import Bot, BotAccount, HandlerCollector, constants, lifespan_wrapper
+from botx.models.stickers import StickerPackFromList
+
+
+@respx.mock
+@pytest.mark.asyncio
+@pytest.mark.mock_authorization
+async def test__iterate_by_sticker_packs__iterate_by_page_succeed(
+    httpx_client: httpx.AsyncClient,
+    host: str,
+    bot_id: UUID,
+    bot_account: BotAccount,
+) -> None:
+    # - Arrange -
+    endpoint = respx.get(
+        f"https://{host}/api/v3/botx/stickers/packs",
+        headers={"Authorization": "Bearer token"},
+        params={"user_huid": "d881f83a-db30-4cff-b60e-f24ac53deecf"},
+    ).mock(
+        return_value=httpx.Response(
+            HTTPStatus.OK,
+            json={
+                "status": "ok",
+                "result": {
+                    "packs": [
+                        {
+                            "id": "26080153-a57d-5a8c-af0e-fdecee3c4435",
+                            "name": "Sticker Pack 1",
+                            "preview": "https://cts-host/uploads/sticker_pack/26080153-a57d-5a8c-af0e-fdecee3c4435/9df3143975ad4e6d93bf85079fbb5f1d.png?v=1614781425296",
+                            "public": True,
+                            "stickers_count": 2,
+                            "stickers_order": [
+                                "a998f599-d7ac-5e04-9fdb-2d98224ce4ff",
+                                "25054ac4-8be2-5a4b-ae00-9efd38c73fb7",
+                            ],
+                            "inserted_at": "2020-11-28T12:56:43.672163Z",
+                            "updated_at": "2021-02-18T12:52:31.571133Z",
+                            "deleted_at": None,
+                        },
+                        {
+                            "id": "89152263-2484-4e00-bc6c-90003027e39e",
+                            "name": "Sticker Pack 2",
+                            "preview": "https://cts-host/uploads/sticker_pack/26080153-a57d-5a8c-af0e-fdecee3c4435/9df3143975ad4e6d93bf85079fbb5f1d.png?v=1614781425296",
+                            "public": True,
+                            "stickers_count": 1,
+                            "stickers_order": [
+                                "a998f599-d7ac-5e04-9fdb-2d98224ce4ff",
+                            ],
+                            "inserted_at": "2020-11-28T12:56:43.672163Z",
+                            "updated_at": "2021-02-18T12:52:31.571133Z",
+                            "deleted_at": None,
+                        },
+                    ],
+                    "pagination": {
+                        "after": None,
+                    },
+                },
+            },
+        ),
+    )
+    built_bot = Bot(
+        collectors=[HandlerCollector()],
+        bot_accounts=[bot_account],
+        httpx_client=httpx_client,
+    )
+    sticker_pack_list = []
+
+    original_sticker_packs_per_page = constants.STICKER_PACKS_PER_PAGE
+    constants.STICKER_PACKS_PER_PAGE = 1  # type: ignore
```

Comment:
Check with STICKER_PACKS_PER_PAGE=2 and 3 sticker packs please

Reply:
Work in both case👌